assets/src/karma-fixture/client_with_context.html

```diff
@@ -12,11 +12,11 @@
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
   <link
-    href="https://fonts.googleapis.com/css?family=Google+Sans|Google+Sans:b&display=swap"
+    href="https://fonts.googleapis.com/css?family=Google+Sans|Google+Sans:b|Google+Sans:500"
```

Comment:
Removed the `display=swap` on purpose? I think we should keep it.

Reply:
It wasn't part loading like this in dashboard. So I making load the same way in both places. 
https://github.com/google/web-stories-wp/pull/6324/commits/96c02aff82c07aae37a4676af04566e9271a0ee1

Reply:
even better!